src/vis/vis.js

```diff
@@ -428,19 +428,24 @@ var Vis = View.extend({
     });
 
     if (!options.skipMapInstantiation) {
-      this.instantiateMap();
+      this._instantiateMap();
     }
 
     return this;
   },
 
-  instantiateMap: function (invalidateMapSize) {
+  /**
+   * Force a map instantiation.
+   * Only expected to be called if {skipMapInstantiation} flag is set to true when vis is created.
+   */
+  instantiateMap: function () {
+    this._instantiateMap();
+    this.mapView.invalidateSize();
+  },
+
```

Comment:
I like the idea but for me it is a bit confusing to have two methods with almost the same name.